src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -383,6 +389,43 @@ private void AddAnonymousTypeName(INamedTypeSymbol symbol)
             }
         }
 
+        /// <summary>
+        /// Returns true if tuple type syntax can be used to refer to the tuple type without loss of information.
+        /// For example, it cannot be used when extension tuple is using non-default friendly names. 
+        /// </summary>
+        /// <param name="symbol"></param>
+        /// <returns></returns>
+        private bool CanUseTupleTypeName(INamedTypeSymbol symbol)
+        {
+            INamedTypeSymbol currentUnderlying = symbol.TupleUnderlyingType;
+
+            if (currentUnderlying.Arity == TupleTypeSymbol.RestPosition)
```

Comment:
Will simplify in the next change. Thanks.

---
In reply to: [62067987](https://github.com/dotnet/roslyn/pull/11047#discussion_r62067987) [](ancestors = 62067987)